vrtUtil/ImageDiff.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const { PNG } = require('pngjs');
+const pixelmatch = require('pixelmatch');
+
+function readPngImage(image) {
+  return new Promise((resolve, reject) => {
+    fs.createReadStream(image).pipe(new PNG())
+      .on('parsed', function onParse() {
+        resolve(this);
+      })
+      .on('error', reject);
+  });
+}
+
+function fixPngImage(image, width, height) {
```

Comment:
Can we rename this function to clarify what it's doing or add comments to describe what it's doing?